fix_it.sh

```diff
@@ -76,7 +76,7 @@ proccess_create_constraint() {
   local TABLE=$1
   local FIELD_RAW=$2
 
-  local PARSE_FIELD="^(CONSTRAINT[^,]*),?$"
+  local PARSE_FIELD="^(CONSTRAINT[^,\n]*),?$"
```

Comment:
it shouldn't be `\\n` since you're in a `"..."` isn't the `\` escaping the `n` when you want `\n` unescaped?

Reply:
I tested it works with `\n`